doc/security/vulnerabilities.rst

```diff
@@ -11,15 +11,380 @@ sufficient time to fix them.  Because the release notes are locked to
+
+- `Zephyr project bug tracker ZEPSEC-29
+  <https://zephyrprojectsec.atlassian.net/browse/ZEPSEC-29>`_
+
+- `PR23646 fix for v1.14.2
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23646>`_
+
+- `PR23649 fix for branch from v2.1.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23649>`_
+
+- `PR23304 fix for v2.2.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23304>`_
+
+CVE-2020-10024
+--------------
+
+ARM Platform Uses Signed Integer Comparison When Validating Syscall
+Numbers
+
+The arm platform-specific code uses a signed integer comparison when
+validating system call numbers. An attacker who has obtained code
+execution within a user thread is able to elevate privileges to that
+of the kernel.
+
+See NCC-ZEP-001
+
+This has been fixed in releases v1.14.2, and v2.2.0, and in a branch
+from v2.1.0,
+
+- `CVE-2020-10024 <https://docs.zephyrproject.org/latest/security/vulnerabilities.html#CVE-2020-10024>`_
+
+- `Zephyr project bug tracker ZEPSEC-30
+  <https://zephyrprojectsec.atlassian.net/browse/ZEPSEC-30>`_
+
+- `PR23535 fix for v1.14.2
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23535>`_
+
+- `PR23498 fix for branch from v2.1.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23498>`_
+
+- `PR23323 fix for v2.2.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23323>`_
+
+CVE-2020-10027
+--------------
+
+ARC Platform Uses Signed Integer Comparison When Validating Syscall
+Numbers
+
+An attacker who has obtained code execution within a user thread is
+able to elevate privileges to that of the kernel.
+
+See NCC-ZEP-001
+
+This has been fixed in releases v1.14.2, and v2.2.0, and in a branch
+from v2.1.0.
+
+- `CVE-2020-10027 <https://docs.zephyrproject.org/latest/security/vulnerabilities.html#CVE-2020-10027>`_
+
+- `Zephyr project bug tracker ZEPSEC-35
+  <https://zephyrprojectsec.atlassian.net/browse/ZEPSEC-35>`_
+
+- `PR23500 fix for v1.14.2
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23500>`_
+
+- `PR23499 fix for branch from v2.1.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23499>`_
+
+- `PR23328 fix for v2.2.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23328>`_
+
+CVE-2020-10028
+--------------
+
+Multiple Syscalls In GPIO Subsystem Performs No Argument Validation
+
+Multiple syscalls with insufficient argument validation
+
+See NCC-ZEP-006
+
+This has been fixed in releases v1.14.2, and v2.2.0, and in a branch
+from v2.1.0.
+
+- `CVE-2020-10028 <https://docs.zephyrproject.org/latest/security/vulnerabilities.html#CVE-2020-10028>`_
+
+- `Zephyr project bug tracker ZEPSEC-32
+  <https://zephyrprojectsec.atlassian.net/browse/ZEPSEC-32>`_
+
+- `PR23733 fix for v1.14.2
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23733>`_
+
+- `PR23737 fix for branch from v2.1.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23737>`_
+
+- `PR23308 fix for v2.2.0 (gpio patch)
```

Comment:
just remove `(gpio patch)`. v2.2.0 received patches for kscan and gpio, but I don't think it is necessary this info in v1.14.2 anyway.

Reply:
Isn't the distinction needed for v2.2.0 because there is a single PR that groups both patches. Only one of them applies to 1.14.x, though.

Reply:
Why don't you just point to backport pr ? https://github.com/zephyrproject-rtos/zephyr/pull/23733